tests/MongoDB.Driver.Core.Tests/Core/Clusters/MultiServerClusterTests.cs

```diff
@@ -451,30 +448,6 @@ public void ProcessDnsResults_should_ignore_empty_end_points_list()
             }
         }
 
-        [Theory]
```

Comment:
this test is related to the removed code from here: https://github.com/DmitryLukyanov/mongo-csharp-driver/pull/51/files#diff-346021f0b586b815ec28dc10028a0a5aL544

Reply:
Ack